lib/alchemy/engine.rb

```diff
@@ -27,6 +27,7 @@
 require_relative './cache_digests/template_tracker'
 require_relative './config'
 require_relative './configuration_methods'
+require_relative './config_loader'
```

Comment:
I don't like this name. Let's be more descriptive: `definitions_loader`, since it's purpose of this class: Loading the yml definitions for pages, elements and cells. The config is something we don't want to tackle here in this PR.

BTW: You did not use this in the `lib/page_layout.rb` file, why?